optax/_src/alias.py

```diff
@@ -1334,6 +1334,90 @@ def optimistic_gradient_descent(
   )
 
 
+def optimistic_adam(
+    learning_rate: base.ScalarOrSchedule,
+    optimism: Optional[float] = None,
+    b1: float = 0.9,
+    b2: float = 0.999,
+    eps: float = 1e-08,
+    eps_root: float = 0.0,
+    mu_dtype: Optional[Any] = None,
+    *,
+    nesterov: bool = True,
+  ) -> base.GradientTransformation:
+  """The Optimistic Adam optimizer.
+
+  This is an optimistic version of the Adam optimizer. It addresses the issue
+  of limit cycling behavior in training Generative Adversarial Networks and
+  other saddle-point min-max problems.
+
```

Comment:
it would be great to have the mathematical description of the algorithm as we have for adagrad, adam, lbfgs, etc.